### ISSUE TITLE
When in SAFE mode and using PlantUML diagrams, rendered images might not show in preview 

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#789]
## Observed vs. expected behavior

When in SAFE mode and using PlantUML diagrams, rendered images will show in preview 

## Steps to reproduce

* change Asciidoctor rendering to SAFE mode in plugin's settings
* change the `imagesdir` attribute to a path that is not the current folder
* add a diagram

```
= Sample with embedded PlantUML

:imagesdir: abc

[plantuml,embedded-plantuml-sample1,svg]
----
@startuml
actor me
control asciidoctor
entity "generated HTML"

me -> asciidoctor : calls
@enduml
----
```

=> then no preview image shows

## Environment

Plugin Version: 0.33.3

This was first reported by @ascheman

**Comment by ahus1** [comment]
When performing the setup shown above, there is a small warning that is rendered at the beginning of the document telling "path is outside of jail; recovering automatically"

The IDE tries to redirect the generated images to a temporary folder outside of the project's directory. In SAFE mode this will not work due to the restrictions of SAFE mode and Asciidoctor Diagram will fall back to the document's current folder. 

The plugin could skip setting that temporary folder and should expect the temporary files to appear in the current folder. Alternatively it could use a temporary folder that is a sub-folder to the document.

**Comment by ahus1** [comment]
This will be fixed in the next upcoming version (0.33.8)
